### PR TITLE
[SERVICE-584] Refresh Service READMEs

### PR DESCRIPTION
**Pull request by bryangaleopenfin**
For ease of following feedback, please can any cross-project feedback go in - https://github.com/HadoukenIO/notifications-service/pull/115

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,31 @@
...
 
 ## Overview
-OpenFin FDC3 provides an implementation of the [FINOS FDC3](https://fdc3.finos.org/) standards for OpenFin-based applications.
+
+The OpenFin FDC3 Service provides an implementation of the [FINOS FDC3](https://fdc3.finos.org/) standards for OpenFin-based applications.
 
 This project consist of 3 parts:
-1. The FDC3 Service, taking care of intents, context and resolving them (UI)
-2. The FDC3 Client, exposing APIs for applications to handle/raise intents with contexts
+1. The FDC3 Provider, taking care of intents, context and resolving them (UI)
```
Comment (ottah) [inline README.md]: Maybe switch context & intents around as you resolve intents, not contexts?

File: README.md [hunk 2]
```diff
@@ -35,9 +35,20 @@ To ensure the service is running, you must declare it in your application config
    {"name": "fdc3"}
 ]
 ```
+During development, you can add a URL for specifying a custom location or a specific version:
 
-### Import the Client API
+```
```
Comment (ottah) [inline README.md]: If you add `json` it will syntax highlight it.

Comment (bryangaleopenfin) [reply README.md]: ** mind-blown-gif **

✅ Done for all of the projects

File: README.md (hunk 1 above)
Comment (bryangaleopenfin) [reply README.md]: ✅

File: README.md [hunk 3]
```diff
@@ -2,68 +2,77 @@
...
 
 ## Overview
-OpenFin FDC3 provides an implementation of the [FINOS FDC3](https://fdc3.finos.org/) standards for OpenFin-based applications.
+
+The OpenFin FDC3 Service provides an implementation of the [FINOS FDC3](https://fdc3.finos.org/) standards for OpenFin-based applications.
 
 This project consist of 3 parts:
-1. The FDC3 Service, taking care of intents, context and resolving them (UI)
-2. The FDC3 Client, exposing APIs for applications to handle/raise intents with contexts
+1. The FDC3 Provider, taking care of contexts and intents, including the intent resolution UI
```
Comment (tomrobertsOF) [inline README.md]: "taking care of" feels a bit vague and informal. Can we do something like "The FDC3 Provider, handling contexts, intents, and the intent resolution UI."

Comment (bryangaleopenfin) [reply README.md]: ✅

**Review by tomrobertsOF — APPROVED** [review]

**Review by ottah — APPROVED** [review]

**Pull request merged**